src/components/Match/MatchGraph.jsx

```diff
@@ -49,9 +49,12 @@ const drawGraphs = (props, id) => {
       },
       tooltip: {
         contents(d, defaultTitleFormat, defaultValueFormat, color) {
```

Comment:
I think this function can be removed completely, the only reason we implemented it was for the sort